.storybook/main.js

```diff
@@ -10,4 +10,8 @@ module.exports = {
   features: {
     postcss: false,
   },
+  managerWebpack: (config, options) => {
```

Comment:
what does this, and is this really needed?

Reply:
I accidentally pushed this one, sorry.

this is our workaround against crashing storybook.

https://github.com/storybookjs/storybook/issues/13795